homeassistant/components/zwave/__init__.py

```diff
@@ -758,26 +774,16 @@ def _object_id(self):
     def device_state_attributes(self):
         """Return the device specific state attributes."""
         attrs = {
-            const.ATTR_NODE_ID: self._value.node.node_id,
+            const.ATTR_NODE_ID: self.node_id,
         }
 
-        try:
-            battery_level = self._value.node.get_battery_level()
-        except RuntimeError:
-            # If we get an runtime error the dict has changed while
-            # we was looking for a value, just do it again
-            battery_level = self._value.node.get_battery_level()
-
-        if battery_level is not None:
-            attrs[ATTR_BATTERY_LEVEL] = battery_level
-
-        location = self._value.node.location
+        if self.battery_level:
```

Comment:
battery_level and wakeup_interval could be 0, but those would still be legal values.

Use `if self.battery_level is not None`